src/main/java/fi/fmi/avi/model/taf/TAF.java

```diff
@@ -23,4 +32,10 @@
 
     Optional<TAFReference> getReferredReport();
 
+    boolean isCancelMessage();
+
+    boolean isMissingMessage();
```

Comment:
This case is different from `isCancelMessage` case [above](https://github.com/fmidev/fmi-avi-messageconverter/pull/68#discussion_r523936410), as missing message is expressed the same way in IWXXM.

Reply:
A small risk I see here when synchronizing from the deprecated `status` (`TAFStatus`). If one temporarily for any reason set status to value `MISSING` and then to something else, we will lose the base forecast. But I doubt this is likely a real world case. And it is only a temporary risk until deprecated `status` is removed. So I'm still voting for my suggestion.

Reply:
This seems like a good approach. What do you mean by "we would lose the base forecast"? Do you mean that setStatus(TAFStatus) would also remove the base forecast and not only that isMissingMessage would be determined from the the presence of the base forecast?

Reply:
Now refactored to determine the missing status from the presence of the baseForecast, both in builder and as the default implementation of the TAF.isMissingMessage() as suggested. The baseForecast is not modified when setting the missing status in the builder , in fact setting the missing status does not have any effect, but it's logged as a warning.

Reply:
What I meant was that invoking `setStatus(MISSING)` would have to clear the base forecast to indicate a missing message in case a missing message is determined from existence of the base forecast instead of a stored boolean property.

Reply:
Fixed in https://github.com/fmidev/fmi-avi-messageconverter/pull/68/commits/53fcba597d421a03444833fd6efe3f4bc74e6a2a to clear the base forecast when calling ```setStatus(MISSING)```.

Note that this causes a condition where a different setter call order results in a different result, see https://github.com/spatineo/fmi-avi-messageconverter/blob/a349c0eb21f162ac8741b5372583f76bb43451bd/src/test/java/fi/fmi/avi/model/taf/immutable/TAFIWXXM21_30_ResilienceTest.java#L75-L81